subsys/net/lib/lwm2m/lwm2m_rw_oma_tlv.c

```diff
@@ -703,7 +708,8 @@ int do_write_op_tlv(struct lwm2m_engine_obj *obj,
 			path->level = 3;
 			ret = do_write_op_tlv_item(context,
 						   &inbuf[tlvpos], len);
-			if (ret < 0) {
+			/* ignore errors for CREATE op*/
+			if (ret < 0 && context->operation != LWM2M_OP_CREATE) {
```

Comment:
Should be
`
if (ret < 0 && (context->operation != LWM2M_OP_CREATE || ret != -ENOTSUP) ) {
   ...
}
`

And I suggest to put a "TODO: support BOOTSTRAP WRITE" here
When the “Write” operation targets an Object or an Object Instance, the LwM2M Client MUST ignore optional resources it does not support in the payload. (5.2.3.4)